_episodes/01-introduction.md

```diff
@@ -6,18 +6,22 @@ questions:
 - "What are containers, and why might they be useful to me?"
 objectives:
 - "Show how software depending on other software leads to configuration management problems."
-- "Identify the problems that software installation problems can pose for research."
-- "Give two examples of how containers can solve software configuration problems." 
+- "Identify the problems that software installation can pose for research."
+- "Explain the advantages virtualization has over traditional computing."
+- "Give two examples of how containers can solve software configuration problems."
```

Comment:
```suggestion
- "Explain how using containers can solve software configuration problems"
```
As I mentioned above, maybe we should change this to remove the "two examples" specificity